newIDE/app/src/ResourcesList/LocalJfxrBridge.js

```diff
@@ -27,27 +27,32 @@ export const openJsfx = ({
     initialResourcePath.lastIndexOf('?cache=')
   );
 
-  const jsfxData = {
+  const jfxrData = {
     resourcePath: initialResourcePath,
     metadata: extraOptions.initialResourceMetadata,
     projectPath,
   };
 
-  ipcRenderer.removeAllListeners('jsfx-changes-saved');
-  ipcRenderer.on('jsfx-changes-saved', (event, newFilePath, fileMetadata) => {
+  ipcRenderer.removeAllListeners('jfxr-changes-saved');
+  ipcRenderer.on('jfxr-changes-saved', (event, newFilePath, fileMetadata) => {
+
+    const resourceName = path.relative(projectPath, newFilePath);// we could make a generic createNewResource utility function that piskel can also use?
```

Comment:
Why not! You can create a function "createOrUpdateResource" (because this function is not only creating a new resource but also "updating" (by removing) the existing one) and put it in a file called `ResourceUtils.js` in `app/src/ResourcesList`. (not in app/src/Utils because I think it's only to be used in ResourcesList folder).

Reply:
Can I try this when revisiting piskel? :) I want to first find out why #720 is happening :)

Reply:
Ok, fair enough. Remove the comment or add a `// TODO: [explanation about the todo]` so that it's easily searchable in the codebase TODOs :)

Reply:
good idea :) 
I think that having a generic function do this can avoid a lot of code repetition, but it must absolutely be done in a way that works best. I noticed that the resource manager uses something similar to load resources - so I could revisit that as well when revisiting piskel